parachain/pallets/inbound-queue/Cargo.toml

```diff
@@ -30,6 +30,8 @@ xcm = { git = "https://github.com/paritytech/polkadot.git", branch = "master", d
 snowbridge-core = { path = "../../primitives/core", default-features = false }
 snowbridge-ethereum = { path = "../../primitives/ethereum", default-features = false }
 snowbridge-router-primitives = { path = "../../primitives/router", default-features = false }
+snowbridge-beacon-primitives = { path = "../../primitives/beacon", default-features = false }
+snowbridge-ethereum-beacon-client = { path = "../../pallets/ethereum-beacon-client", default-features = false }
```

Comment:
See my previous comment, we don't want the pallet to directly depend on the beacon light client crate.

You can move these to `dev-dependencies` to though, as its fine to depend on these for tests.